views/partials/header.njk

```diff
@@ -33,3 +35,17 @@
         {% endif %}
     </div>
 </header>
+
+ {% if phaseName %}
+    <div class="outer-phase-banner">
+
+        {{ govukPhaseBanner({
+            tag: {
+                text:  phaseName,
+                classes: phaseNameColour
+            },
+            html: 'This test version of DPS contains real data which may be up to two weeks old.'
```

Comment:
Is this messaged also shown in dev?

Reply:
yes same message for both DEV and PRE-PROD